kong/db/strategies/cassandra/connector.lua

```diff
@@ -301,4 +301,28 @@ function CassandraConnector:truncate()
 end
 
 
+function CassandraConnector:truncate_table(table_name)
+  local ok, err = self:connect()
+  if not ok then
+    return nil, err
+  end
+
+  local cql = string.format("TRUNCATE TABLE %s.%s",
+                            self.keyspace, table_name)
+
+  local ok, err = self:query(cql)
```

Comment:
I believe we should specify this query as a write operation, as per the 4th argument of this function?